src/pages/policy/PolicyRightSidebar.jsx

```diff
@@ -93,6 +100,277 @@ function TimePeriodSelector(props) {
   );
 }
 
+/**
+ * Selector like the full/lite toggle that toggles between 'static' and 'dynamic' versions of the dataset.
+ * @param {Object} props
+ * @param {Object} props.policy
+ * @param {Object} props.metadata
+ * @returns {React.ReactElement}
+ */
+function BehavioralResponseToggle(props) {
+  const { policy, metadata } = props;
+
+  const dateString = `${defaultStartDate}.${String(defaultForeverYear)}-12-31`;
+  const behavioralResponseReforms = useMemo(
```

Comment:
any way to pull this up by reform id instead?

Reply:
Doing so would make manipulating the resulting policies more challenging, require an extra fetch, and would require us to hunt out the correct policy if we ever want to alter these inputs. While it can be done, I think it would be less efficient overall.